image/image_linux.go

```diff
@@ -417,6 +419,21 @@ func setupSeed(tsto *ToolingStore, model *asserts.Model, opts *Options) error {
 		if err := installCloudConfig(rootDir, gadgetUnpackDir); err != nil {
```

Comment:
the comment before the block probably needs expanding:

// early config & cloud-init config (done at install for Core 20)